tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -160,13 +207,33 @@ Check GPU Resources
         ${elements}=  Get WebElements      //article
         ${len}=  Get Length    ${elements}
         Should Be Equal As Integers     ${len}  1
-        Page Should Contain     //article[@id="python-gpu-numba-tutorial"]
-        Page Should Contain     //a[@href="https://github.com/ContinuumIO/gtc2018-numba"]
+        Page Should Contain Element     //article[@id="python-gpu-numba-tutorial"]
+        Page Should Contain Element     //a[@href="https://github.com/ContinuumIO/gtc2018-numba"]
     ELSE
         ${elements}=  Get WebElements      //article
         ${len}=  Get Length    ${elements}
         Should Be Equal As Integers     ${len}  0
-        Page Should Not Contain     //article[@id="python-gpu-numba-tutorial"]
-        Page Should Not Contain     //a[@href="https://github.com/ContinuumIO/gtc2018-numba"]
+        Page Should Not Contain Element     //article[@id="python-gpu-numba-tutorial"]
+        Page Should Not Contain Element     //a[@href="https://github.com/ContinuumIO/gtc2018-numba"]
     END
 
+Select Checkbox Using Id
+    [Documentation]    Select check-box
+    [Arguments]    ${id}
+    Select Checkbox    id=${id}
+    sleep    10s
+
+Deselct Checkbox Using Id
+    [Documentation]    Deselect check-box
+    [Arguments]    ${id}
+    Click Element    id=${id}
```

Comment:
why not using `[Unselect Checkbox](https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Unselect%20Checkbox)` to be consistent with `Select Checkbox Using Id`?